devito/passes/clusters/blocking.py

```diff
@@ -45,6 +49,9 @@ def process(self, clusters):
 
         processed = super(Blocking, self).process(processed)
 
+        if self.skewing:
```

Comment:
This is now what I suggested on Slack, and putting it here doesn't make much sense to me (nested Queues?)

Reply:
Fixed